CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # CHANGELOG for mcrouter
 
+## [v3.0.0] (2017-08-15)
+
+### Breaking changes:
+* Removed support for Ubuntu 14.04
+* Converted `node['folly']['version']` to `node['mcrouter']['commit_hash']` given that FB has a weird release process.
+* Removed `node['folly']['version']` given that the commit has is part of mcrouter project. See [FOLLY_COMMIT](https://github.com/facebook/mcrouter/blob/release-36-0/mcrouter/FOLLY_COMMIT)
```

Comment:
I think what I meant is "removed the ability to specify the Folly commit hash as an attribute" does that make sense?